README.md

```diff
@@ -66,7 +66,7 @@ buffer and drop them when necessary.
 
 The notify state is enabled by the user by calling the PO Box. Its sole
 task is to verify if there is any message in the buffer. If there is, it
-will respond to the PO Box's owner with a `{mail, new_data}` message
+will respond to the PO Box's owner with a `{mail, BoxPid, new_data}` message
```

Comment:
Is this a new backwards incompatible change, or the doc was always wrong?

Reply:
Looks like that. I don't recall changing it. I will double check.

Reply:
Okay so I found when it was changed commit 9325f4abafccb5dba7e697bbf3856b8eccb22ac5 . Which was commited by you on 2013-08-14 09:31:17.

Reply:
Sounds good, so I just had not updated the doc/messed up versioning back then.